### ISSUE TITLE
Howto change *_redo.lua IF statement, that the "Farming Redo" Mod is supported

### ISSUE THREAD

**Issue by JohannesPaulHoffmann** [acmgit/cucina_vegana#28]
Hi,

in order to support the "Farming Redo" Mod, the individual Plant *_redo.lua files can be modified to set (the variable) germ to = "_1", when the case of 0 takes apply.

The Plants get then, when Placed, not Placed as Seeds anymore, but as *_1 State, where "Farming Redo" can work with.

Regards,
Johannes

P.S.: I added Plants to the growing Group, on several locations ("Mapgen, "crop_def", in the 'items.lua' file, "minetest.register_node({})", but brought no effect, but "Farming Redo"s README shows that.

**Comment by acmgit** [comment]
Yes i know this, this is the reason, why i added in the settingtypes "cucina_vegana.plant_settings.germ_launch (Start germ @) int 0".
This is by default 0, change this setting and the crops start at another stage, for example 1.

The problem is, minetest without farming_redo start without problems with stage 0.
With farming_redo installed, you need to set this to 1.

Next problem is, minetest without farming_redo, the seeds named: seed_xxxxx. (xxxx is the name of the crop.
In farming_redo, the seeds are named: xxxxx_seed.

Than, minetest's declaration of new crops works with one declaration, where you add the number of steps.
In farming_redo, you have to define each stage separate.

Afaik the group "growing" has no effect in the vanilla minetest.

**Comment by tenplus1** [comment]
@JohannesPaulHoffmann - I've already done that in this fork.

https://codeberg.org/tenplus1/cucina_vegana
